lib/db.js

```diff
@@ -54,10 +54,11 @@ class LimitDBRedis extends EventEmitter {
     this.globalTTL = (config.globalTTL || ms('7d')) / 1000;
 
     const redisOptions = {
+      // a low commandTimeout value would likely cause sharded clusters to fail `enableReadyCheck` due to it running `CLUSTER INFO`
+      // which is a slow command. timeouts are being handled by the client#dispatch method.
       enableOfflineQueue: false,
       keyPrefix: config.prefix,
       password: config.password,
-      commandTimeout: config.commandTimeout || DEFAULT_COMMAND_TIMEOUT,
```

Comment:
Is there going to be any timeout on initial connection?

Reply:
If you timeout the initial connection, you fail to establish a proper connection and it will never recover.

Reply:
Cool, just want to make sure we don't hang forever trying to connect 👍